perllib/FixMyStreet/Cobrand/Gloucester.pm

```diff
@@ -162,4 +162,125 @@ sub open311_post_send {
     }
 }
 
+=head2 _asset_layer_mapping
+
+Gloucester's Alloy integration has assets separated into different layers.
+When looking up a parent asset for a report that doesn't have one, we need
+to know which layer(s) to check. This method provides a mapping from the
+report category to the relevant asset layer(s). The asset layer names
+correspond to asset layers configured in the Tilma proxy service.
+
+=cut
+
+sub _asset_layer_mapping {
+    return {
```

Comment:
We need to make sure this list is up to date with latest category changes.

Reply:
Can we get this from the configuration? A script like this:
```
use FixMyStreet::Cobrand;
use Data::Dumper;
use JSON::MaybeXS;
my $c = FixMyStreet::Cobrand::Gloucester->new;
my $layers = $c->feature("asset_layers");
$layers = [ @$layers ];
shift @$layers;
foreach (@$layers) {
	my $copy = { %$_ };
	foreach (@{$copy->{asset_category}}) {
		push @{$out->{category}{$_}}, $copy->{http_options}{params}{layer};
	}
	if ($copy->{asset_group}) {
		$_ = $copy->{asset_group};
		push @{$out->{group}{$_}}, $copy->{http_options}{params}{layer};
	}
}
print JSON->new->utf8->canonical->pretty->encode($out);
```
Seems to produce the same lookup as here (with category name rather than "email"), if I've understood the layers correctly